pkg/plugin/plugin.go

```diff
@@ -175,10 +175,6 @@ func (p *V1Plugin) Decrypt(ctx context.Context, request *pb.DecryptRequest) (*pb
 
 	result, err := p.svc.Decrypt(input)
 	if err != nil {
-		select {
-		case p.healthCheck.healthCheckErrc <- err:
```

Comment:
what about when we fail encrypt https://github.com/kubernetes-sigs/aws-encryption-provider/blob/24ffc614b97e06b8fdcf62de2aedad0f9f3b5e7e/pkg/plugin/plugin.go#L93?

I think the fix should be at a higher level to only report health check failure if ALL plugins fail health check https://github.com/kubernetes-sigs/aws-encryption-provider/blob/master/pkg/healthz/healthz.go#L22